### ISSUE TITLE
[50.0] The unit Clan Medium Battle Armor (Naval) 'Bär' (Sqd5) could not be found

### ISSUE THREAD

**Issue by IllianiCBT** [MegaMek/megamek#5681]
### Environment
Version: Standalone, 50.0
OS: Mac OS X 14.5
Java: Eclipse Adoptium 17.0.11

### Description
Launching mm shows the following errors in log:

```
09:57:17,121 ERROR [megamek.client.generator.RandomUnitGenerator] {Random Unit Generator unit populator}
megamek.client.generator.RandomUnitGenerator.readRat(RandomUnitGenerator.java:216) - The unit Clan Medium Battle Armor (Naval) 'Bär' (Sqd5) could not be found in the Clan Ghost Bear Secondline Infantry (WoRS) RAT (Ghost Bear Secondline Infantry.txt)

09:57:17,147 ERROR [megamek.client.generator.RandomUnitGenerator] {Random Unit Generator unit populator}
megamek.client.generator.RandomUnitGenerator.readRat(RandomUnitGenerator.java:216) - The unit Clan Medium Battle Armor (Naval) 'Bär' (Sqd5) could not be found in the 3145 Rasalhague Dominion Battle Armor - Second Line RAT (BA Second Line.txt)

09:57:17,380 ERROR [megamek.client.generator.RandomUnitGenerator] {Random Unit Generator unit populator}
megamek.client.generator.RandomUnitGenerator.readRat(RandomUnitGenerator.java:216) - The unit Clan Medium Battle Armor (Naval) 'Bär' (Sqd5) could not be found in the 3085 Ghost Bear Dominion Battlearmor Second Line RAT (3085 Ghost Bear Dominion Battlearmor Second Line.txt)

09:57:17,981 ERROR [megamek.client.generator.RandomUnitGenerator] {Random Unit Generator unit populator}
megamek.client.generator.RandomUnitGenerator.readRat(RandomUnitGenerator.java:216) - The unit Clan Medium Battle Armor (Naval) 'Bär' (Sqd5) could not be found in the Rasalhague Dominion Battle Armor - 3135+ RAT (Battle Armor.txt)
```

### Files
[megamek.log](https://github.com/user-attachments/files/16111172/megamek.log)

**Comment by HammerGS** [comment]
Should be fixed with https://github.com/MegaMek/mekhq/commit/557a8e8a4c1cce98d1cd355b974b9c3fdaaa854b

**Comment by IllianiCBT** [comment]
> Should be fixed with [MegaMek/mekhq@557a8e8](https://github.com/MegaMek/mekhq/commit/557a8e8a4c1cce98d1cd355b974b9c3fdaaa854b)

Issue persists

```
10:39:25,157 ERROR [megamek.client.generator.RandomUnitGenerator] {Random Unit Generator unit populator}
megamek.client.generator.RandomUnitGenerator.readRat(RandomUnitGenerator.java:216) - The unit Clan Medium Battle Armor (Bär) could not be found in the Clan Ghost Bear Secondline Infantry (WoRS) RAT (Ghost Bear Secondline Infantry.txt)

10:39:25,176 ERROR [megamek.client.generator.RandomUnitGenerator] {Random Unit Generator unit populator}
megamek.client.generator.RandomUnitGenerator.readRat(RandomUnitGenerator.java:216) - The unit Clan Medium Battle Armor (Bär) could not be found in the 3145 Rasalhague Dominion Battle Armor - Second Line RAT (BA Second Line.txt)

10:39:25,349 ERROR [megamek.client.generator.RandomUnitGenerator] {Random Unit Generator unit populator}
megamek.client.generator.RandomUnitGenerator.readRat(RandomUnitGenerator.java:216) - The unit Clan Medium Battle Armor (Bär) could not be found in the 3085 Ghost Bear Dominion Battlearmor Second Line RAT (3085 Ghost Bear Dominion Battlearmor Second Line.txt)

10:39:25,843 ERROR [megamek.client.generator.RandomUnitGenerator] {Random Unit Generator unit populator}
megamek.client.generator.RandomUnitGenerator.readRat(RandomUnitGenerator.java:216) - The unit Clan Medium Battle Armor (Bär) could not be found in the Rasalhague Dominion Battle Armor - 3135+ RAT (Battle Armor.txt)
```
